src/api/bank-accounts/bank-account-requests.md

```diff
@@ -0,0 +1,114 @@
+| createdBy                | {% dt CRN %}       | The User or API Key that created the Bank Account Request.                                 |
+| updatedAt                | {% dt Timestamp %} | When the Bank Account Request was updated.                                                 |
+| updatedBy                | {% dt CRN %}       | The User or API Key that updated the Bank Account Request.                                 |
+| authorizationRedirectUrl | String             | A redirect url to begin the OAuth 2 flow.                                                  |
+
+### Bank Account Request Types
+
+|     Field      |                                   description                                    |
+| :------------- | :------------------------------------------------------------------------------- |
+| quartz-details | Request will initiate an Oauth 2 flow for authorizing access to account details. |
+| quartz-payment | Request will initiate an Oauth 2 flow for authorizing access to create payments. |
+
+## Operations
+
+### Authorize Bank Account Request **EXPERIMENTAL**
+{% reqspec %}
+  POST '/api/bank-account-requests'
+  auth 'api-key'
+  example {
+    body ({
+      accountId: 'uooxSens6ykJaim1Cu1Q55',
+      type: 'quartz-details',
+    })
+  }
+{% endreqspec %}
+
+{% h4 Required Fields %}
+
+|   Field   |  Type  |            Description            |
+| :-------- | :----- | :-------------------------------- |
+| accountId | String | The id of the owning [Account][]. |
+| type      | String | The type of Bank Account Request. |
+
+{% h4 Optional Fields %}
+
+|     Field     |  Type  |                                    Description                                    |
+| :------------ | :----- | :-------------------------------------------------------------------------------- |
+| bankAccountId | String | The id of the associated [Bank Account][]. Required if type is `quartz-payment`. |
+
+{% h4 Example response payload %}
+
+{% json %}
+  id: 3KVjuKW2CZCJeJVqPxwkX7
+  accountId: B4u4WZCu3joZFVWT3XjWW3
+  type: quartz-details
+  status: created
+  createdAt: 2022-03-31T02:56:29.713Z
+  createdBy: crn:B4u4WZCu3joZFVWT3XjWW3:api-key:MyApiKey
+  updatedAt: 2022-03-31T02:56:29.713Z
+  updatedBy: crn:B4u4WZCu3joZFVWT3XjWW3:api-key:MyApiKey
+  "authorizationRedirectUrl": "https://example.com/authorization-redirect-url"
+{% endjson %}
+
+### Create Bank Account Request **EXPERIMENTAL**
+{% reqspec %}
+  POST '/api/bank-account-requests/{bankAccountRequestId}/authorize'
+  path_param 'bankAccountRequestId', '3KVjuKW2CZCJeJVqPxwkX7'
+  auth 'api-key'
+  example {
+    body ({
+      code: 'hKpKupTM391pE10xfQiorMxXarRKAHRhTfH_x',
+    })
+  }
+{% endreqspec %}
+
+{% h4 Required Fields %}
+
+| Field |  Type  |                      Description                      |
+| :---- | :----- | :---------------------------------------------------- |
+| code  | String | authorization code returned returned from third party |
```

Comment:
Would be good to just clean this up by capitalising the description and adding a full stop.

Reply:
```suggestion
| code  | String | Authorization code returned returned from third party. |
```